umpleonline/ump/GeometricSystem.ump

```diff
@@ -115,143 +115,154 @@ class NonPolyhedra{
 }
 class ArbitraryPolygon{
 	isA Polygon;
-	double getArea(){/*implementation */}
-	double getPerimeterLength(){/*implementation */}
-	boolean isLessThan(Polygon object){}
-	boolean isEqual(Polygon object){}	
+	double getArea(){return 1.0;}
+	double getPerimeterLength(){return 1.0;}
+	boolean isLessThan(Polygon object){return true;}
+	boolean isEqual(Polygon object){return true;}
+	boolean isEqual(CurvedShape object){return true;}	
 }
 
 class Sphere{
 	isA NonPolyhedra;
 	double radius;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}
 }
 
 class Torus {
 	isA NonPolyhedra;
 	double radius_r; 
-	double radiusr_R;
+	double radius_R;
 	double theta;
 	double phi;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}
 }
 
 class Cylinder{
 	isA NonPolyhedra;
 	double radius;
 	double height;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}	
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}	
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}
 }
 
 class Cone{
 	isA NonPolyhedra;
 	double radius;
 	double height;
 	double sideLength;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}			
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}			
 }
 
 class Cube{
 	isA Polyhedra;
 	double edgeLength;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}		
-	boolean isEqual(Polyhedra object){/*implementation */}
-	boolean isLessThan(Polyhedra object){/*implementation */}	
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}		
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
```

Comment:
Looks like the following method should be deleted.
`boolean isEqual(CurvedShape object){return true;}`